src/GitHub.VisualStudio/UI/Views/PullRequestDetailView.xaml.cs

```diff
@@ -306,5 +306,15 @@ void ViewCommentsClick(object sender, RoutedEventArgs e)
             }
             catch { }
         }
+
+        void OpenHyperlink(object sender, ExecutedRoutedEventArgs e)
+        {
+            var url = e.Parameter.ToString();
+
+            if (!string.IsNullOrEmpty(url))
+            {
+                VisualStudioBrowser.OpenUrl(new Uri(url));
```

Comment:
Oh actually sorry, they're not created by us here. If they're malformed they'll be passed to the browser malformed, as they would on .com I imagine.